WalkingChallenge/Challenge/JourneyCard.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  JourneyCard.swift
+//  WalkingChallenge
+//
+//  Created by Mohsin on 24/02/19.
+//  Copyright © 2019 AKDN. All rights reserved.
+//
+
+import Foundation
+import SnapKit
+import Foundation
+import UIKit
+
+typealias StylizedJourneyView =  JourneyCardViewCell & LayoutableCardView
```

Comment:
Why do you need a new type alias?  Can you not extend the `StylizedCarView` appropriately?